indent/nim.vim

```diff
@@ -196,6 +196,11 @@ function! GetNimIndent(lnum) abort
             let result = -1
           endif
         endif
+	  elseif line =~# '=\s*if.*:$'
+		" if expression. Nim's `if` can be not only statements but also expressions.
+		" let a = if cond:
+		"   "a"
+		let result += shiftwidth()
```

Comment:
This section of the code is for triggering indentation when the `:` symbol is typed. As such, it wouldn't handle indentation for what happens in the line after that.

I've just tried the expression you commented here but it seems to be indented just fine. The generic code for handling this case is in lines 222 -- 245.

If it's not working for some other expressions, please let me know and I can help you figure out where it should be handled.

Reply:
Hmm? I made a PR because this example didn't work, but I just tried it and it works. Thanks.